packages/@okta/vuepress-site/docs/reference/api/groups/index.md

```diff
@@ -1853,7 +1853,9 @@ Specifies required and optional properties for a Group. The `objectClass` of a G
 
 #### ObjectClass: okta:user_group
 
-Profile for any Group that is not imported from Active Directory
+Profile for any Group that is not imported from Active Directory. Specifies the [standard](#default-profile-properties) and [custom](#custom-profile-properties) profile properties for a Group.
+
+##### Default Profile Properties
 
 | Property    | Description              | DataType | Nullable | Readonly | MinLength | MaxLength | Validation |
```

Comment:
Readonly > Read-only

Reply:
Thanks Christine. While I agree with you that it should be "Read-only". This title is used in many property tables. I'd like to verify with the team before doing a mass search-and-replace on this item.